### ISSUE TITLE
No email address in Code of Conduct

### ISSUE THREAD

**Issue by The-Compiler** [PyCQA/meta#13]
From `source/code-of-conduct.rst`:

> instances of abusive, harassing, or otherwise unacceptable behavior may be
> reported by contacting the project team at **[INSERT EMAIL ADDRESS]**.

**Comment by sigmavirus24** [comment]
Correct. We never established who should be receiving these emails. Ideally it is more than one person.

I know so far, for volunteers, we have:

- me
- @IanLee1521 

And that's it. I'd like at least one more just to be safe. That said, since most projects in the organization will be using this particular version, I'm not certain we want to keep that in there. Instead I'm wondering if we should remove that from that copy and have each project say something like:

```rst
This project follows the `PyCQA code of conduct <...>`_. Instances of abusive,
harassing, or otherwise unacceptable behaviour may be reported by contacting
the project team at [INSERT EMAIL ADDRESS(ES)].
```

**Comment by IanLee1521** [comment]
IMO, I think that the approach of spelling it out in each project who to contact is probably the right one.

**Comment by The-Compiler** [comment]
FWIW, if we do add a list of mail addresses, I'd be happy to volunteer (Florian Bruhin, me@the-compiler.org) if you feel like that'd be a good fit. If not, that's fine as well.

**Comment by sigmavirus24** [comment]
Well three people is better than two. @saifelse are you comfortable adding these three email addresses to the list?

**Comment by sigmavirus24** [comment]
We need to get this wrapped up. RIght now we have 3 volunteers, we can always add more. Is anyone willing to send a PR to add them?

**Comment by hugovk** [comment]
Please see PR https://github.com/PyCQA/meta/pull/29.
